### DISCUSSION TITLE
Change ytdl-core to youtube-dl

### DISCUSSION THREAD

**Discussion by diespinozah** [eritislami/evobot#1343] [Ideas]
I've been testing the bot changing the youtube video downloader because ytdl-core crashed many times with long songs (more than 30m).

The performance of youtube-dl is much better and has no crashes. 

Many people on the issues branch has been complaining againts the issue mentioned before and that change for me is the best tested solution.

**Comment by eritislami** [comment]
Thanks will look into this, currently working on migrating to Discord.js 14 & slash commands with response buttons and all the new stuff, I'm way behind on all the new changes and features that have been added to discord.js so not sure about the timeline of everything but I'm excited to be more involved again

**Reply by diespinozah** [reply]
If you wanna test the changes, just text me and i'll gladly share it with you

**Reply by Moini-Rimboo** [reply]
> If you wanna test the changes, just text me and i'll gladly share it with you

Can you Share me the new code with youtube-dl please?

**Reply by diespinozah** [reply]
> Thanks will look into this, currently working on migrating to Discord.js 14 & slash commands with response buttons and all the new stuff, I'm way behind on all the new changes and features that have been added to discord.js so not sure about the timeline of everything but I'm excited to be more involved again

You can use play-dl, it works fine too

**Reply by diespinozah** [reply]
> > If you wanna test the changes, just text me and i'll gladly share it with you
> 
> Can you Share me the new code with youtube-dl please?

I'm sorry for I was on my vacations, I don't know how to merge the change to the new code. I could share you the 2.5.1 version, but It's not currently working

**Comment by diespinozah** [comment]
Hi @eritislami, I gladly contribute my part to this repository.

I made a pull request changing ytdl-core to play-dl, I've testing it and it works.

You can test it and then if you wanna add it to the repository it would be an honor.

note: I deleted on package-json the ytdl-core and ytdl-core-discord dependencies. If you think that the dependencies should not be deleted let me know and I just do another pull without deleting them.
